rfp-proposals/RFP提案：“MCM system”.md

```diff
@@ -0,0 +1,93 @@
+
+# Open Grant Proposal: `Project Title`
+
+**Name of Project:**Miner Cluster Management system
+
+**Proposal Category:**  `app-dev`
+
+**Proposer:** `Jonney hu`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:**  "Yes" 
+
+# Project Description
+
+- MCM system provides a complete set of management solutions for the mine and mine pool that are not ready yet, so as to make adequate preparations for Filcoin that will be launched soon. For each mine, mine pool to solve how to manage node mining machine, monitoring node mining machine, how to distribute Filecoin and other services, including mining machine management module, mining machine safety monitoring module, data summary module, FIL collection module, distribution module.
```

Comment:
This is a great idea, but I think a bit too early. As you correctly point out, mining and mining pools aren't ready yet, so it would be great to wait for later waves when those designs are more settled.